Backend objects: Updated Objects Player: new constructor, Team: Find()

evangboucher opened the pull request:
Player has a new constructor that only takes a string.

Team has a find player method, one method for finding a player with a string name and the other for finding a player object.

Commit:
Update README for demo.

evangboucher committed:
updated Objects: Player new const. Team: find()

Added a new constructor for the Player objects.
You can now construct a player with just a name (string).
The position is currently not being used (may not get used at all, and
will be removed later if found not useful).

The Team object has two new methods for linearly searching for
a player either by a string name or by a player object (Must be a
literal object: IE, objects must be the same literal object).

DanMorganiOS commented on android/src/com/fireball/teampicker/Team.java:
```diff
@@ -73,10 +82,59 @@ public void setName(String name) {
 	 * @param players the players to set
+=======
+	 * @param players Array List of players to set
+>>>>>>> Stashed changes
```
Need to resolve these and re-add.

evangboucher committed:
Updated the Objects to reflect Mark's comments

Updated objects. Find player method for team. New player constructor.
Edited team test.
